package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "scripts": {
     "build": "tsc",
-    "dev": "start-storybook -p 4442",
+    "dev": "NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 4442",
```

Comment:
storybook wouldn't boot without this for me I'm afraid, happy to remove it if it boots for you 🙏